module1/homework/grayscaleImages/compression.hpp

```diff
@@ -0,0 +1,24 @@
+// #pragma once
+
+// #include <array>
+// #include <vector>
+// #include <utility>
+
+// constexpr size_t width = 240;
+// constexpr size_t height =160; 
+
+// std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(const std::array<std::array<uint8_t, width>, height>& image);
+// std::array<std::array<uint8_t, width>, height> decompressGrayscale(std::vector<std::pair<uint8_t, uint8_t>> comressedImage);
```

Comment:
Generally we avoid keeping old code commented,because it would became garbage after long time working on project. We are using version control system like git if we would  to access the last working version.

Reply:
Remove if not necessary :)